.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# These owners will be the default owners for everything in the repo. Unless a
+
+# later match takes precedence, the team will be requested for review when
+
+# someone opens a pull request.
+
+cli/ @grafbase/backend
```

Comment:
should we create a `cli` team that could include outside collaborators eventually?

Reply:
We could yeah, but for review purposes I think backend makes sense no?